lib/jobs/ansible-job.js

```diff
@@ -78,10 +95,23 @@ function ansibleJobFactory(
      * @memberOf AnsibleJob
      */
      AnsibleJob.prototype._cleanup = function cleanup() {
+        var self = this;
+        try {
+            var data = fs.readFileSync(self.outFileName, 'utf8');
```

Comment:
The synchronous version API is only used in unit-testing, if there is no strong reason that we have to use the synchronous API,  otherwise we need use the async API for better performance concern.